gtsfm/averaging/rotation/cycle_consistency.py

```diff
@@ -30,6 +31,25 @@
 MAX_INLIER_MEASUREMENT_ERROR_DEG = 5.0
```

Comment:
I think 5 deg is a bit small if we’re using median, split among 3 edges, that is less than 2 degrees per edge. I think 3 degrees per edge is better.  But yes, that also means we could accept edges that have an error of 9 degrees in the worst case, although that is unlikely because most edges would have cycles with other noisy edges.

Reply:
that's fair. i guess it's a precision/recall tradeoff. I think I prefer to err more on the side of precision than recall. let me compare the metrics some more with both thresholds.